src/lib/seam/components/AccessCodeDetails/AccessCodeDetails.tsx

```diff
@@ -41,18 +42,54 @@ export function AccessCodeDetails({
     )
   }
 
+  const filterCodeErrors = () => {
+    return (
+      accessCode.errors.filter((error) => {
+        if (!error?.is_access_code_error) return true
```

Comment:
Should fix the linter
```suggestion
        if (error?.is_access_code_error !== true) return true
```